portal-web/docroot/html/taglib/ui/search_iterator/list.jsp

```diff
@@ -274,7 +274,7 @@ if (iteratorURL != null) {
 				}
 			%>
 
-				<td class="table-cell <%= columnClassName %>" colspan="<%= entry.getColspan() %>">
+				<td class="table-cell-content <%= columnClassName %>" colspan="<%= entry.getColspan() %>">
```

Comment:
Hey @fernandosouza 

I think we might be able to remove this class. Usually the `table-cell-content` class is used when wanting to make one cell larger than the others in it's column, so adding it to all of them is probably unnecessary. Was there another reason you added it that I'm not aware of?